hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rolloutgrouptargets/RolloutGroupTargetsListGrid.java

```diff
@@ -238,11 +239,18 @@ public String convertToPresentation(final Status status, final Class<? extends S
         }
 
         private String processActionStatus(final Status status) {
-            final StatusFontIcon statusFontIcon = statusIconMap.get(status);
+            final StatusFontIcon statusFontIcon = isDownloadOnlyRollout() ? statusIconMap.get(Status.FINISHED)
```

Comment:
Please check if this mapping is right. Doesn't this mean that in a Download Only rollout no matter what status the target is in, the UI will always display the finished icon? (Possibly the mapping is already done when the target reports its status and this method can be left untouched)